client/lib/Browser.ts

```diff
@@ -200,7 +200,12 @@ export async function createBrowser(
   };
   const coreClientSession = await coreClient.createSession(sessionOptions);
 
-  const { sessionsDataLocation } = coreClientSession;
-  scriptInstance.launchReplay(sessionName, sessionsDataLocation);
+  let showReplay = envShowReplay;
+  if (options.showReplay !== undefined) showReplay = options.showReplay;
+  if (showReplay) {
+    scriptInstance.launchReplay(sessionName, coreClientSession);
+  }
   return new Browser(coreClientSession, sessionName);
 }
+
+const envShowReplay = Boolean(JSON.parse(process.env.SA_SHOW_REPLAY ?? 'true'));
```

Comment:
Why create this `envShowReplay` const? It's only used once, on line 203. Why not just move this value to 203?

Regardless, it seems a bit odd to declare a const at the bottom.

Reply:
Just because it has to parse the string to translate to a Boolean, so it seemed like it made sense to read it in once. You find that odd?

Reply:
In some circumstances I wouldn't, but in this case it's only being used in the createBrowser method, which isn't a commonly used method (i.e., like a `log` method that might be called thousands of times in the course of a single script). It complicates the code for a trivial (practically non-existent) efficiency boost.